packages/runtime/src/state/react-builder-preview.ts

```diff
@@ -550,14 +551,6 @@ export function messageChannelMiddleware(
             window.getSelection()?.removeAllRanges()
             break
 
-          case ActionTypes.SET_LOCALE: {
-            const { pathname: currentPathname, query } = Router
-            const pathname = (action.payload.pathname ?? currentPathname).replace(/^\//, '/')
-
-            Router.replace({ pathname, query }, undefined, { locale: action.payload.locale })
-            break
-          }
```

Comment:
Now that I'm looking at this closely, I don't think this code path was every being called 🤔